recipe-server/client/control_new/less/pages/gateway.less

```diff
@@ -0,0 +1,31 @@
+.page-gateway {
+  .anticon {
+    display: block;
+    font-size: 3em;
+    margin-bottom: 0.5em;
+    text-align: center;
+  }
+
+  .gw-col {
+    background: #E1F4CB;
```

Comment:
After discussion and some testing, we actually don't need to do any of the Webpack business. I've converted this over to rely on Ant's color variables.